src/components/settings/SettingsSSOTab/SSOFormWorkflowConfig.tsx

```diff
@@ -1,37 +1,106 @@
-import type { FormWorkflowStep } from '../../forms/FormWorkflow';
-import SSOConfigConnectStep from './steps/NewSSOConfigConnectStep';
+import omit from 'lodash/omit';
+
+import type { FormWorkflowHandlerArgs, FormWorkflowStep } from '../../forms/FormWorkflow';
+import SSOConfigConnectStep, { validations as SSOConfigConnectStepValidations } from './steps/NewSSOConfigConnectStep';
 import SSOConfigConfigureStep from './steps/NewSSOConfigConfigureStep';
-import SSOConfigAuthorizeStep from './steps/NewSSOConfigAuthorizeStep';
+import SSOConfigAuthorizeStep, { validations as SSOConfigAuthorizeStepValidations } from './steps/NewSSOConfigAuthorizeStep';
 import SSOConfigConfirmStep from './steps/NewSSOConfigConfirmStep';
+import LmsApiService from '../../../data/services/LmsApiService';
+import handleErrors from '../utils';
 
-type SSOConfigCamelCase = {};
+type SSOConfigSnakeCase = {
+  uuid?: string,
+  enterprise_customer: string,
+  is_removed: boolean,
+  active: boolean,
+  identity_provider: string,
+  metadata_url: string,
+  metadata_xml: string,
+  entity_id: string,
+  update_from_metadata: boolean,
+  user_id_attribute: string,
+  full_name_attribute: string,
+  last_name_attribute: string,
+  email_attribute: string,
+  username_attribute: string,
+  country_attribute: string,
+  submitted_at: null,
+  configured_at: null,
+  validated_at: null,
+  odata_api_timeout_interval: null,
+  odata_api_root_url: string,
+  odata_company_id: string,
+  sapsf_oauth_root_url: string,
+  odata_api_request_timeout: null,
+  sapsf_private_key: string,
+  odata_client_id: string,
+  oauth_user_id: string,
+  sp_metadata_url?: string
+};
 
-export const SSOFormWorkflowConfig = () => {
+export const SSOFormWorkflowConfig = ({ enterpriseId }) => {
   const placeHolderButton = (buttonName?: string) => () => ({
     buttonText: buttonName || 'Next',
     opensNewWindow: false,
     onClick: () => {},
   });
 
-  const steps: FormWorkflowStep<SSOConfigCamelCase>[] = [
+  const saveChanges = async ({
+    formFields,
+    errHandler,
+  }:FormWorkflowHandlerArgs<SSOConfigSnakeCase>) => {
+    let err = null;
+    let updatedFormFields = formFields;
+    const submittedFormFields: SSOConfigSnakeCase = omit(formFields, ['idpConnectOption', 'is_pending_configuration', 'sp_metadata_url']);
+    submittedFormFields.enterprise_customer = enterpriseId;
+    if (formFields?.uuid) {
+      try {
+        const updateResponse = await LmsApiService.updateEnterpriseSsoOrchestrationRecord(
```

Comment:
If I had to make an executive decision.... I say `Sso` > `SSO` but I have not followed that rule up until now 🙈

Reply:
If I was starting fresh I would follow the `Sso` > `SSO` rule(to better handle names which have different acronyms back-to-back), but given that ~80% of the existing admin portal codebase does the opposite, is it worth the amount of refactoring it would take to bring it into line?

Reply:
Per discussion with @alex-sheehan-edx offline, going to leave as-is for now, as we should try and use the `Sso` going forward, but renaming all the existing `SSO` acronyms to it now is more trouble than it's worth.